packages/viewer/src/lib/interactions/multiTouchInteractionHandler.ts

```diff
@@ -48,8 +44,13 @@ export abstract class MultiTouchInteractionHandler
           Matrix2.dot(previousToCurrent)
         )
       );
+      const center = Point.create(
+        (this.currentPosition1.x + this.currentPosition2.x) / 2,
+        (this.currentPosition1.y + this.currentPosition2.y) / 2
+      );
```

Comment:
Should this actually be looking at `point1` and `point2`? Looks like this may mean that the center point we compute is always one behind with this setup